docs/developer-docs/latest/setup-deployment-guides/installation/cli.md

```diff
@@ -63,6 +63,12 @@ yarn create strapi-app my-project
 
 4. (Custom installation type only) Name your project's database.
 
+::: tip
+Experimental Strapi versions are released every day. You can create a new Strapi application based on the latest experimental release using `npx create-strapi-app@experimental`.
```

Comment:
Every Tuesday through Saturday at midnight GMT (I think it'd GMT else it's PST, we will find out tonight when GitHub actions kicks off).

Reply:
Thanks! Fixed it (I looked at the logs, seems to kick in at midnight GMT)